rules_jvm_export/jvm_export/support/deploy.py

```diff
@@ -0,0 +1,264 @@
+#!$PYTHON_PATH
+
+# Originally from https://github.com/vaticle/bazel-distribution/blob/master/maven/templates/deploy.py
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import print_function
+from xml.etree import ElementTree
+
+import argparse
+import hashlib
+import os
+import re
+import shutil
+import subprocess as sp
+import sys
+import tempfile
+from pathlib import Path
+from posixpath import join as urljoin
+from urllib.parse import urlparse
+
+USAGE_STR = """
+bazel run <target>.publish -- release
+"""
+
+MAVEN_REPOS = {
+    "snapshot": "{snapshot}",
+    "release": "{release}",
+    "local": "file:~/.m2/repository/",
+}
+
+
+def main():
+    args = _parse_args()
+    repo_type = "local" if args.local else args.command
+    maven_url = MAVEN_REPOS[repo_type]
+    jar_path = "$JAR_PATH"
```

Comment:
To make future maintenance easier, I would suggest declaring global constants at the beginning of the file with the values from the substitution and using them throughout the rest of the script.

Reply:
Fixed in ae4ff41